src/components/SkyBrowser/SkybrowserTabs.jsx

```diff
@@ -0,0 +1,422 @@
+import ScrollOverlay from '../common/ScrollOverlay/ScrollOverlay';
+import CenteredLabel from '../common/CenteredLabel/CenteredLabel';
+import Row from '../common/Row/Row'
+import NumericInput from '../common/Input/NumericInput/NumericInput'
+import ColorPickerPopup from '../common/ColorPicker/ColorPickerPopup'
+import Checkbox from '../common/Input/Checkbox/Checkbox'
+
+import { excludeKeys } from '../../utils/helpers';
+import TooltipSkybrowser from './TooltipSkybrowser';
+import SkybrowserFocusEntry from './SkybrowserFocusEntry';
+import styles from './SkybrowserTabs.scss';
+import { Resizable } from 're-resizable';
+
+class SkybrowserTabs extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            height: 185,
+            currentHeight: 185,
+            showButtonInfo: [false, false, false, false, false],
+            showSettings: false,
+        }
+        this.createTabs = this.createTabs.bind(this);
+        this.createImageList = this.createImageList.bind(this);
+        this.onResizeStop = this.onResizeStop.bind(this);
+        this.onResize = this.onResize.bind(this);
+        this.setRef = this.setRef.bind(this);
+        this.showTooltip = this.showTooltip.bind(this);
+        this.hideTooltip = this.hideTooltip.bind(this);
+        this.createSettings = this.createSettings.bind(this);
+        this.valueToColor = this.valueToColor.bind(this);
+        this.toggleFaceCamera = this.toggleFaceCamera.bind(this);
+        this.toggleRadiusAzimuthElevation = this.toggleRadiusAzimuthElevation.bind(this);
+        this.setOpacityOfImage = this.setOpacityOfImage.bind(this);
+        this.removeImageSelection = this.removeImageSelection.bind(this);
+        this.createButtons = this.createButtons.bind(this);
+        this.toggleShowSettings = this.toggleShowSettings.bind(this);
+    }
+
+    get inheritedProps() {
+        const doNotInclude = 'closeCallback className position size title';
+        return excludeKeys(this.props, doNotInclude);
+    }
+
+    get position() {
+        if (!this.wrapper) return { top: '0px', left: '0px' };
+        const { top, right} = this.wrapper.getBoundingClientRect();
+        return { top: `${top}`, left: `${right}`};
+    }
+
+    setRef(what) {
+        return (element) => {
+          this[what] = element;
+        };
+    }
+
+    showTooltip(i) {
+    let showButtonInfoNew= [...this.state.showButtonInfo];
+    let selected = {...showButtonInfoNew[i]};
+    showButtonInfoNew[i] = true;
+    this.setState({showButtonInfoNew});
+        this.setState({showButtonInfo : showButtonInfoNew});
+    }
+
+    hideTooltip(i) {
+      let showButtonInfoNew= [...this.state.showButtonInfo];
+      let selected = {...showButtonInfoNew[i]};
+      showButtonInfoNew[i] = false;
+      this.setState({showButtonInfoNew});
+          this.setState({showButtonInfo : showButtonInfoNew});
+      }
+
+    toggleFaceCamera() {
```

Comment:
This function would benefit from using deconstruction on props: 
```
const { isFacingCamera, selectedBrowser , selectedTarget } = this.props;
const uriBrowser = 'ScreenSpace.' + selectedBrowser + '.FaceCamera';
const uriTarget = 'ScreenSpace.' + props.selectedTarget + '.FaceCamera';

this.props.api.setPropertyValueSingle(uriBrowser, !isFacingCamera);
this.props.api.setPropertyValueSingle(uriTarget, !isFacingCamera);
```

Reply:
Could also use templated strings for the URI: `ScreenSpace.${selectedBrowser}.FaceCamera`;

Makes the strings easier to read, IMO :)